Tests/test_PDB_parse_pdb_header.py

```diff
@@ -0,0 +1,50 @@
+"""Unit tests for PARTS of the parse_pdb_header module of Bio.PDB"""
+
+import unittest
+import warnings
+
+try:
+    import numpy
+    from numpy import dot  # Missing on old PyPy's micronumpy
+    del dot
+    from numpy.linalg import svd, det  # Missing in PyPy 2.0 numpypy
+except ImportError:
+    from Bio import MissingPythonDependencyError
+    raise MissingPythonDependencyError(
+        "Install NumPy if you want to use Bio.PDB.")
+
+
+from Bio.PDB.parse_pdb_header import parse_pdb_header, _parse_remark_465
+
+
+class ParseReal(unittest.TestCase):
+    """Testing with real PDB file(s)."""
+    def test_parse_pdb_with_remark_465(self):
+        """
+        Tests that parse_pdb_header now can identify some REMARK 465 entries.
+        """
+        header = parse_pdb_header("PDB/2XHE.pdb")
+        self.assertTrue(header["has_missing_residues"])
+        self.assertEqual(len(header["missing_residues"]), 142)
+        self.assertIn( {"model":None, "res_name":"GLN", "chain":"B", "ssseq":267, "insertion":None}  , header["missing_residues"])
+        header = parse_pdb_header("PDB/1A8O.pdb")
+        self.assertFalse(header["has_missing_residues"])
+        self.assertEqual(header["missing_residues"], [])
+    def test_parse_remark_465(self):
+        "A UNIT-test for the private function _parse_remark_465"
```

Comment:
Please use tripe double-quotes as per PEP257, and include trailing full stop. e.g.

```
"""Test private function _parse_remark_465 for missing residues."""
```